docs/protocol/reference/deployed.md

```diff
@@ -32,45 +32,31 @@ Rinkeby (LPT) is the set of contracts that originally implemented the alpha prot
 - PollCreator: [0xBf824EDb6b94D9B52d972d5B25bCc19b4e6E3F3C](https://etherscan.io/address/0xbf824edb6b94d9b52d972d5b25bcc19b4e6e3f3c)
 - MerkleSnapshot: [0x24ebEd82c681f435E944BEEbFAEEAaE443D08438](https://etherscan.io/address/0x24ebed82c681f435e944beebfaeeaae443d08438)
 
+## Ethereum Rinkeby (Confluence)
+- BondingManager: [0xC40df4db2f99e7e235780A93B192F1a934f0c45b](https://rinkeby.etherscan.io/address/0xC40df4db2f99e7e235780A93B192F1a934f0c45b)
+- BondingManagerProxy: [0xC40df4db2f99e7e235780A93B192F1a934f0c45b](https://rinkeby.etherscan.io/address/0xC40df4db2f99e7e235780A93B192F1a934f0c45b)
```

Comment:
For consistency let's describe contracts with "Proxy" and "Target" suffix the same way that we do elsewhere i.e.:

BondingManager (Proxy)
BondingManager (Target)